back-end-side/Controllers/ReportReader.cs

```diff
@@ -82,7 +104,7 @@ public ReportReader(int bank, IFormFile fileData)
                 {
                     records.ElementAt(i).Amount /= 100;
 
-                    if (records.ElementAt(i).PaymentType.Equals("K"))
+                    if (records.ElementAt(i).PaymentType.Equals("C"))
                     {
                         records.MoveToOtherList(ref IncomeList, i);
```

Comment:
Not your code, but you can change that MoveToOtherList wouldn't ask reference to IncomeList, as list is sent as a reference anw

Reply:
Okay